examples/declutter-group.js

```diff
@@ -0,0 +1,84 @@
+import GeoJSON from '../src/ol/format/GeoJSON.js';
+import Map from '../src/ol/Map.js';
+import VectorLayer from '../src/ol/layer/Vector.js';
+import VectorSource from '../src/ol/source/Vector.js';
+import View from '../src/ol/View.js';
+import {Fill, Stroke, Style} from '../src/ol/style.js';
+import {Group} from '../src/ol/layer.js';
+import {apply} from 'ol-mapbox-style';
+
+const geojsonObject = {
+  'type': 'FeatureCollection',
+  'crs': {
+    'type': 'name',
+    'properties': {
+      'name': 'EPSG:3857',
+    },
+  },
+  'features': [
+    {
+      'type': 'Feature',
+      'geometry': {
+        'type': 'Polygon',
+        'coordinates': [
+          [
+            [-12e6, 3.5e6],
+            [-12e6, 5.5e6],
+            [-10e6, 5.5e6],
+            [-10e6, 3.5e6],
+            [-12e6, 3.5e6],
+          ],
+        ],
+      },
+    },
+  ],
+};
+
+const styles = [
+  new Style({
+    stroke: new Stroke({
+      color: 'rgba(180, 180, 255, 1)',
+      width: 1,
+    }),
+    fill: new Fill({
+      color: 'rgba(200, 200, 255, 0.85)',
+    }),
+  }),
+];
+
+const source = new VectorSource({
+  features: new GeoJSON().readFeatures(geojsonObject),
+});
+
+const overlay = new VectorLayer({
+  source: source,
+  style: styles,
+  startDeclutterGroup: true,
+});
+
+const LayerGroup = new Group();
+apply(
+  LayerGroup,
+  'https://api.maptiler.com/maps/topo-v2/style.json?key=get_your_own_D6rA4zTHduk6KOKTXzGB'
+);
+
+const map = new Map({
+  target: 'map',
+  view: new View({
+    center: [-10203186.115192635, 4475744.563386],
+    zoom: 4,
+  }),
+  layers: [LayerGroup, overlay],
+});
+
+function update(event) {
+  overlay.set('startDeclutterGroup', event.target.value === 'true');
```

Comment:
If you make `startDeclutterGroup` an observable property, you'll still need to add a getter and setter (`getStartDeclutterGroup()` and `setStartDeclutterGroup()`). But I'd just make it a private member property on `Layer`, and add a public getter and setter.

Reply:
This sent me on a reading exercise as I never knew the scope of `set` in regards to being observable. I am going with private member getter/setter, but I'm curious if they should be included in the API docs. Looking at the code, I thought it was based on usefulness outside other classes, but that line is blurry...should I include these in the API docs?

Reply:
If the setter is not part of the API, there's also no need to add a setter in the first place.

Reply:
Our use-case, weather raster layers over vector basemaps, could find the ability to toggle this feature convienent, but perhaps that is not the best way to achieve that behavior? Is that enough reason to keep a setter and add it to the API? Otherwise, a configure and forget wouldn't need a setter.

Reply:
Configure and forget sounds good. A setter can be added later if someone need's it, but I think we're good without.

Reply:
I updated the branch, a couple of thoughts:
1. Without a setter, I had to set the private property directly in the constructor from options--I assume that is best practice
2. There are unrelated browser test failures, curious to see how that pans out in tests ran by the workflow